navigation/index.tsx

```diff
@@ -17,6 +18,8 @@ export type RootStackParamList = {
   SignUp: undefined
   Settings: undefined
   Signaling: undefined
+  UserAvailability: undefined
+  EditAvailability: undefined
```

Comment:
remove `UserAvailability` since it's not needed. and also rename `EditAvailability` to `EditActivity`
```suggestion
  EditActivity: undefined
```